ONSdigital/dp-compose
# Add kafka cluster to dp-compose, improved README and add Makefile

### What

- Create a kafka cluster in docke-compose with 3 kafka nodes and 1 zookeepr
- Added Makefile for convenience and consistency with Cantabular compose

Needed by: Kafka health check to display `WARNING` if enough brokers are healthy, but not all of them. Trello card: https://trello.com/c/NmmQVJGg/5435-kafka-client-fails-health-check-when-msk-security-patches-are-applied-s1

Needed to test the bug fix https://github.com/ONSdigital/dp-kafka/pull/46 locally

### How to review

- Make sure changes make sense
- Try to start/stop the services

### Who can review

Anyone
